src/frequenz/sdk/actor/power_distributing/_component_status.py

```diff
@@ -49,3 +49,14 @@ class ComponentStatusEnum(enum.Enum):
 
     WORKING = 2
     """Component is working"""
+
+
+@dataclass(frozen=True)
```

Comment:
what about making it `kw_only=True` too?

Reply:
the two fields have different types.